Rocket.Chat/Assets.xcassets/Icons/Insecure Image.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "insecureresource@1x.png",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "filename" : "insecureresource@2x.png",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
```

Comment:
Waiting for new images from @arthurgiustina 🤓

Reply:
@arthurgiustina Sent us these ones:

![resource unavailable 3x](https://user-images.githubusercontent.com/551004/35332034-9e17fc2c-00d7-11e8-8228-36c01e646426.png)
![resource unavailable 2x](https://user-images.githubusercontent.com/551004/35332035-9e256894-00d7-11e8-94f0-c9685c0880da.png)
![resource unavailable](https://user-images.githubusercontent.com/551004/35332036-9e33abd4-00d7-11e8-8951-25dd6b1a0eb3.png)

Reply:
But then we need a workaround to show the file always at 100% height